lredis/commands.lua

```diff
@@ -40,6 +42,48 @@ function methods:client_pause(delay)
 	return handle_ok_or_err(resp)
 end
 
+function methods:hmget(key, ...)
+	local resp = self:call("HMGET", key, ...)
+	if type(resp) == "table" then
+		local ret = {}
+		for i, v in pairs({...}) do
+			ret[v]=resp[i]
+		end
+		return ret
+	else
+		return resp
+	end
+end
+
+function methods:hmset(key, tbl)
+	local data = {}
+	for i,v in pairs(tbl) do
```

Comment:
Did you want `pairs` or `ipairs` here?
If you actually wanted `pairs` then use `k` instead of `i`.